runtime/doc/terminal.txt

```diff
@@ -1683,13 +1683,17 @@ Change default signs ~
 Termdebug uses the hex number of the breakpoint ID in the signcolumn to
 represent breakpoints. If it is greater than "0xFF", then it will be displayed
 as "F+", due to we really only have two screen cells for the sign.
+You may also use decimal breakpoint signs instead, in which case IDs greater
+than 99 will be displayed as "9+".
 
-If you want to customize the breakpoint signs: >
+If you want to customize the breakpoint signs to show `>>` in the signcolumn: >
 	let g:termdebug_config['sign'] = '>>'
+If you would like to use decimal (base 10) breakpoint signs: >
+	let g:termdebug_config['sign_decimal'] = 1
 If there is no g:terminal_config yet you can use: >
 	let g:termdebug_config = {'sign': '>>'}
-
-After this, breakpoints will be displayed as `>>` in the signcolumn.
+Or: >
+	let g:termdebug_config = {'sign_decimal': '>>'}
```

Comment:
That looks wrong. A few lines above you are saying to set the `g:termdebug_config['sign_decimal']` to show decimal signs, and now you are saying to use this key to configure to use a completely different sign (that's also not what the code seems to do)

Reply:
Oops - meant to document that you can instantiate the g:termdebug_config that way with sign_decimal. Just pushed a change to fix that